### DISCUSSION TITLE
How to contribute to next-learn documentations update.

### DISCUSSION THREAD

**Discussion by ansulagrawal** [vercel/next.js#57721] [Show and tell]
Hi, I want to contribute to nextjs learn documentain update I tried to find in vercel/next-learn project but not able to find it so can you tell which repo is that and where is the code so I can help to contribute to it.

Thanks in advance.

**Comment by icyJoseph** [comment]
Hi,

I think that one is closed source. In the next-learn repository you find some of the course practical material, that is code starters and such.

If you've see an issue in the Learn page, you can still report it at next-learn though.

**Comment by emapeire** [comment]
Hi,
I would like to contribute too. I found some errors that I would like to report, but Idk who I should contact to enter the linear repo.

**Reply by icyJoseph** [reply]
Mmm that's weird... GitHub converted that into a link... Not what I meant. This is the repository:

- https://github.com/vercel/next-learn

**Reply by emapeire** [reply]
thanks!

**Reply by ansulagrawal** [reply]
But it only contains code for examples not the docs code

**Reply by icyJoseph** [reply]
Yes, like I said, the actual batteries behind the next learn site, are closed source.

**Reply by icyJoseph** [reply]
But, you can open an issue there, reporting something wrong with the learn flow, and then someone will fix it in the actual project.
